.sync.yml

```diff
@@ -1,4 +1,25 @@
 ---
+Gemfile:
+  required:
+    ':system_tests':
+      - gem: 'puppet-module-posix-system-r#{minor_version}'
+        platforms: ruby
+      - gem: 'puppet-module-win-system-r#{minor_version}'
+        platforms:
+          - mswin
+          - mingw
+          - x64_mingw
+      - gem: beaker
+        version: '~> 3.13'
+        from_env: BEAKER_VERSION
+      - gem: beaker-abs
```

Comment:
Do we really need this gem? I don't really see us using this.

Reply:
Just pulled in from PDK templates. I'll fine-tune that later